### PR TITLE
[DOCS] More info on disabling swap

### PR DESCRIPTION
**Pull request by geekpete**
Let users know that disabling swap might create some IO and that an Elasticsearch restart isn't required to complete this process.

### CODE REVIEW

File: docs/reference/setup/sysconfig/swap.asciidoc [hunk 1]
```diff
@@ -28,6 +28,10 @@ On Linux systems, you can disable swap temporarily by running:
 sudo swapoff -a
 --------------
 
+This will cause an amount of IO workload as the contents of swap is copied
```
Comment (danielmitterdorfer) [inline docs/reference/setup/sysconfig/swap.asciidoc]: how about "an amount of IO workload" -> "an I/O spike"?

**Review by danielmitterdorfer — APPROVED** [review]
Looks fine overall. I left a suggestion.

**Comment by jasontedor** [comment]
What are the motivations here? I am concerned this would have the effect to discourage disabling swap yet typically it would be something we want to encourage.

**Comment by geekpete** [comment]
The main point was that a restart wasn't required, but thought it might be useful to explain what's going on when you run this command, that it's going to copy everything out of that swap partition back into memory which means reading it all off disk.
I think we might omit the IO spike mention and I can adjust.

**Comment by danielmitterdorfer** [comment]
I think with the new wording it should be fine. @jasontedor objections?

**Comment by jasontedor** [comment]
No objections here now. Thanks for the iteration.

**Pull request merged**